### PR TITLE
cli: include consistent commands to fix error conditions

### LINKED ISSUE #1733: Apply guidelines from design to error messages with commands that fix
pr: #1790
Estimate: 1 day

### PR DESCRIPTION
**Pull request by elopio**
- [x] Have you followed the [guidelines for contributing](https://github.com/snapcore/snapcraft/blob/master/CONTRIBUTING.md)?
- [x] Have you signed the [CLA](http://www.ubuntu.com/legal/contributors/)?
- [x] If this is a bugfix. Have you checked that there is a bug report open for the issue you are trying to fix on [bug reports](https://bugs.launchpad.net/snapcraft)?
- [x] If this is a new feature. Have you discussed the design on the [forum](https://forum.snapcraft.io)?
- [x] Have you successfully run `./runtests.sh static`?
- [x] Have you successfully run `./runtests.sh unit`?

-----
Closes #1733
Requires #1640

### CODE REVIEW

File: snapcraft/tests/unit/test_errors.py [hunk 1]
```diff
@@ -58,9 +57,8 @@ class ErrorFormattingTestCase(unit.TestCase):
                 "The 'test-step' step of 'test-part' is out of date:\n"
                 "The 'test-property1' and 'test-property2' part properties "
                 "appear to have changed.\n"
-                "To continue, please clean that part's "
-                "'test-step' step by running:\n"
-                "snapcraft clean test-part -s test-step\n")}),
+                "To clean that part's 'test-step', run "
```
Comment (kyrofa) [inline snapcraft/tests/unit/test_errors.py]: This feels a little... jarring to me. Our guidance is "We have a problem. Here's why it's a problem. Here's how you fix it." What we're moving to here seems to be "We have a problem. Here's why it's a problem. Run this command to do X."

It seems a small difference, but there's a disconnect I can't quite explain between "Here's why it's a problem" and "Run this command." It doesn't flow, it seems like we're missing the "here's how you fix it" part, and just throwing a magical command on there. Is that just me?

**Comment by kyrofa** [comment]
Oh come on. Just because I was reviewing a specific commit, github hides them?

File: snapcraft/internal/errors.py [hunk 2]
```diff
@@ -54,9 +54,8 @@ class StepOutdatedError(SnapcraftError):
         'Failed to reuse files from previous build: '
         'The {step!r} step of {part!r} is out of date:\n'
         '{report}'
-        'To continue, please clean that part\'s {step!r} step '
-        'by running:\n'
-        'snapcraft clean {parts_names} -s {step}\n'
+        'To clean that part\'s {step!r} step, run '
+        '`snapcraft clean {parts_names} -s {step}.`'
```
[the remark this answers is not in the document]
Comment (elopio) [reply snapcraft/internal/errors.py]: The style I'm trying to follow is backticks for commands, single quotes for the exception attributes.

File: snapcraft/tests/unit/test_errors.py (hunk 1 above)
Comment (elopio) [reply snapcraft/tests/unit/test_errors.py]: So, like explaining what will happen in english and then give the command that should be run?
I'm not really sure I'm getting your comment, so maybe a suggestion of an error message that follows your proposed style would be useful.

**Pull request merged**